app/views/shared/_cost_of_living_payments.html.slim

```diff
@@ -2,5 +2,10 @@
 - summary_text = t("cost_of_living_summary_and_details.header#{suffix}")
 
 = govuk_details(summary_text:)
-  - t("cost_of_living_summary_and_details.list#{suffix}").each
-    div = _1
+  - if FeatureFlags.enabled?(:mtr_accelerated, without_session_data: true)
+    ul
+      - t("cost_of_living_summary_and_details.list").each do |item|
+        li.govuk-text = item
+  - else
+    - t("cost_of_living_summary_and_details.list_legacy").each do |item|
+      .govuk-text = item
```

Comment:
Am I right in thinking we need to get rid of  

- `= render "shared/cost_of_living_payments"` 

and replace it with  

- `= render "shared/cost_of_living"`

That way we can get rid of this partial and the text in `en.yml`

Reply:
Will be picked up on another ticket.